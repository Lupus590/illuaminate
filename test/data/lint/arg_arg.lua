-- config: (lint (only var:arg-arg))

local function f(arg, arg) end
