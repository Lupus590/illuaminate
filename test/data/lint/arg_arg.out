arg_arg.lua:[3:17-3:20]: Argument named "arg". [var:arg-arg]
   │
 3 │ local function f(arg, arg) end
   │                  ^^^
arg_arg.lua:[3:22-3:25]: Argument named "arg". [var:arg-arg]
   │
 3 │ local function f(arg, arg) end
   │                       ^^^
No errors and 2 warnings
